org.eclipse.scanning.event/src/org/eclipse/scanning/event/ConsumerImpl.java

```diff
@@ -321,6 +325,16 @@ public void run() {
 		
 	}
 	
+	/**
+	 * Awaits the start of the consumer
+	 * @throws InterruptedException 
+	 * @throws Exception
+	 */
+	public void awaitStart() throws InterruptedException {
+		if (latchStart!=null) latchStart.await();
+	}
+
+	
 	private PauseBean getPauseBean(String submissionQueueName) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "getPauseBean" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)